addons/im_livechat/views/im_livechat_chatbot_templates.xml

```diff
@@ -8,7 +8,7 @@
                 <t t-set="head">
                     <script>
                         <t t-call="im_livechat.loader">
-                            <t t-set="info" t-value="{ 'available': True, 'options': { 'init': { 'available_for_me': True, 'rule': { 'chatbot': chatbot_script.id }, 'storeData': storeData }, 'isTestChatbot': True, 'force_thread': channel_data }, 'server_url': server_url }"/>
+                            <t t-set="info" t-value="{ 'available': True, 'options': { 'chatbot_test_store': chatbot_test_store }, 'server_url': server_url }"/>
```

Comment:
Can't we get rid of this page?
Chatbot test should just open a new chat window in backend and everything should work.
A lot of embed code should be core/common instead.

Reply:
I agreed but once again this is out of scope of this PR. This first step is a good step towards this goal but its already a big diff which is quite hard to maintain. Removal of services/chat bot test page & co will come as follow ups.